modules/core/02-client/keeper/migrations.go

```diff
@@ -3,7 +3,7 @@ package keeper
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
-	"github.com/cosmos/ibc-go/v9/modules/core/02-client/migrations/v7"
+	v7 "github.com/cosmos/ibc-go/v9/modules/core/02-client/migrations/v7"
```

Comment:
This is annoying. My editor will always put this in place as well and then the linter complains about a redundant import alias... happens with vim-go and vscode extensions... 🫠

Reply:
ill play with the linter settings